src/python/pants/backend/project_info/rules/dependencies.py

```diff
@@ -25,7 +25,7 @@ class DependencyType(Enum):
 
 # TODO(#8762) Get this rule to feature parity with the dependencies task.
 class DependenciesOptions(LineOriented, GoalSubsystem):
-    """Print the target's dependencies."""
+    """List the transitive dependencies of the input targets."""
```

Comment:
Not necessarily transitive. We default to `--no-transitive`.

Reply:
Hmmm, the help text for `--transitive` is wrong, it looks like:

```"Run dependencies against transitive dependencies of targets specified on the command line."``` 

That sounds like "run this goal on every target in the closure". But looking at how the option is used in the code, I think this should say 

```
"Show all transitive dependencies. If unspecified, show only direct dependencies."
```

I'll make that change (and to the goal description) but please confirm my understanding.

Reply:
Ah, yes, your read is correct. `--filedeps-transitive` could be improved too to reflect this. `core/project_info/filedeps.py`.